pupil-spa/src/app/auditEntry.ts

```diff
@@ -1,9 +1,33 @@
-export type AuditEntryType = 'CheckStarted' | 'QuestionRendered' | 'QuestionAnswered' | 'PauseRendered' | 'UserInput';
```

Comment:
do you think it would be better to have one class per file instead of having multiple classes within one?